### DISCUSSION TITLE
Discovery server and fastdds spy

### DISCUSSION THREAD

**Discussion by Renzobc** [eProsima/Fast-DDS-spy#49] [General]
- setting the FASTRTPS_DEFAULT_PROFILES_FILE environmental variable to a configuration pointing to a discovery server.
 - fastddsspy only discovers the participants that are the discovery servers themselfs.

**Comment by juanlofer-eprosima** [comment]
Hi @Renzobc 

I am moving the issue to discussions as what you are trying is currently not supported.

As of today, Fast-DDS-spy is only able to sniff local communications; this is, a participant using multicast discovery is created on a user-defined DDS domain, and only messages on this domain are captured and displayed.
In addition, configuration of Fast-DDS-spy through XML profiles is neither supported at the moment.

Feel free to contact support@eProsima.com if interested in adding support for any new features.

Cheers
